mapping_parenting_tech/analysis/prototyping/play_store/get_play_store_apps.py

```diff
@@ -0,0 +1,146 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.3
+#   kernelspec:
+#     display_name: 'Python 3.8.12 64-bit (''mapping_parenting_tech'': conda)'
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Code to extract app ids from [list of] web pages saved from Google Play store
+
+# %% [markdown]
+# Do imports and set file locations
+
+# %%
+from pathlib import Path
+from google_play_scraper import Sort, app, reviews_all
+from tqdm import tqdm
+import re
+import json
+import sys
+
+INPUT_PATH = Path("~/github/mapping_parenting_tech/inputs/data/")
+OUTPUT_PATH = Path("~/github/mapping_parenting_tech/outputs/data/")
+STORE_PATH = "play_store/"
+PLAY_DATA_FILE = "playstore_innerHTML.html"
+OUTPUT_PLAY_IDS_FILE = "play_store_ids.json"
+OUTPUT_PLAY_DETAILS_FILE = "play_store_details.json"
+OUTPUT_PLAY_REVIEWS_FILE = "play_store_reviews.json"
+
+input_data = INPUT_PATH.expanduser() / STORE_PATH / PLAY_DATA_FILE
+
+if input_data.exists():
+    print("Ok to proceed")
+else:
+    print("Didn't find file")
+
+
+# %% [markdown]
+# ## For Play Store
+# Read HTML and extract list of app ids
+
+# %%
+html_doc = open(input_data, "r")
+html = html_doc.read()
+html_doc.close
```

Comment:
This could be simplified as:
```suggestion
with open(input_data, "r") as infile:
    html = infile.read()
```

Reply:
`infile.close`?